build/kits/jboss-as7/modules/io/swagger/main/module.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module xmlns="urn:jboss:module:1.0" name="io.swagger">
+
+	<properties>
+        <property name="jboss.api" value="private"/>
+    </properties>
+    
+    <resources>
+        <resource-root path="swagger-annotations-${version.swagger-jaxrs}.jar" />
+        <resource-root path="swagger-core-${version.swagger-jaxrs}.jar" />
+        <resource-root path="swagger-jaxrs-${version.swagger-jaxrs}.jar" />
+        <resource-root path="swagger-models-${version.swagger-jaxrs}.jar" />
+        <resource-root path="jackson-dataformat-xml-${version.com.fasterxml.jackson.swagger}.jar" />
+        <resource-root path="jackson-dataformat-yaml-${version.com.fasterxml.jackson.swagger}.jar" />
```

Comment:
Shouldn't all the "jackson" dependencies need to their own module, if there is already module, we need to create a new version of that module.